docs/accounts/Basics.md

```diff
@@ -31,6 +31,23 @@ Each of these will have their own requirements for opening, which will be checke
 <!-- theme: warning -->
 > SIPPs are currently in a private beta. If you wish to access them, please contact your account manager for more details
 
+## Account currencies
+
+Each account created within the system can have an allocated currency. The currencies currently supported are GBP, EUR and USD. Once the currency of an account has been set, it cannot be changed. Any [Portfolios](docs/guides/adc4b06b3d4ed-portfolio-basics) created under an account will be opened in the same currency as the account itself. 
```

Comment:
I'm being a little nitpicky here but you mention that an account has an "allocated" currency, which makes it sound like we are always deciding for the tenant. I would maybe drop the word "allocated"
```suggestion
Each account created within the system can have a currency. The currencies currently supported are GBP, EUR and USD. Once the currency of an account has been set, it cannot be changed. Any [Portfolios](docs/guides/adc4b06b3d4ed-portfolio-basics) created under an account will be opened in the same currency as the account itself. 
```